ssc_interface_wrapper_ros2/src/converter_worker.cpp

```diff
@@ -0,0 +1,391 @@
+
+        velocity_accel_sub_ = create_subscription<automotive_platform_msgs::msg::VelocityAccelCov>("as/velocity_accel_cov",10,
+                                                                    std::bind(&Converter::velocity_accel_cb, this, std_ph::_1));
+        curvature_feedback_sub_ = create_subscription<automotive_platform_msgs::msg::CurvatureFeedback>("as/curvature_feedback", 10,
+                                                                    std::bind(&Converter::curvature_feedback_cb, this, std_ph::_1));
+        throttle_feedback_sub_ = create_subscription<automotive_platform_msgs::msg::ThrottleFeedback>("as/throttle_feedback", 10,
+                                                                    std::bind(&Converter::throttle_feedback_cb, this, std_ph::_1));
+        brake_feedback_sub_ = create_subscription<automotive_platform_msgs::msg::BrakeFeedback>("as/brake_feedback", 10,
+                                                                    std::bind(&Converter::brake_feedback_cb, this, std_ph::_1));
+        gear_feedback_sub_ = create_subscription<automotive_platform_msgs::msg::GearFeedback>("as/gear_feedback", 10, 
+                                                                    std::bind(&Converter::gear_feedback_cb, this, std_ph::_1));
+        steering_wheel_sub_ = create_subscription<automotive_platform_msgs::msg::SteeringFeedback>("as/steering_feedback", 10,
+                                                                    std::bind(&Converter::steering_feedback_cb, this, std_ph::_1));
+                                                                                                                                                                                                                                                                                                                                                                                                                       
+        callback_from_ssc_feedbacks(velocity_feedback_, curvature_feedback_, throttle_feedback_, brake_feedback_, gear_feedback_, steering_feedback_);
+        callback_for_twist_update(velocity_feedback_,curvature_feedback_,steering_feedback_);
+
+        // Setup publishers
+        // To autoware 
+        vehicle_status_pub_ = create_publisher<autoware_msgs::msg::VehicleStatus>("vehicle_status",10);
+        current_twist_pub_ = create_publisher<geometry_msgs::msg::TwistStamped>("vehicle/twist",10);
+        // To SSC
+        steer_mode_pub_ = create_publisher<automotive_platform_msgs::msg::SteerMode>("as/arbitrated_steering_commands", 10);
+        speed_mode_pub_ = create_publisher<automotive_platform_msgs::msg::SpeedMode>("as/arbitrated_speed_commands",10);
+        turn_signal_pub_ = create_publisher<automotive_platform_msgs::msg::TurnSignalCommand>("as/turn_signal_command",10);
+        gear_pub_ = create_publisher<automotive_platform_msgs::msg::GearCommand>("as/gear_select", 1);
```

Comment:
In ros1 this is latched. I believe the way to duplicate that behavior here would be 
```
rclcpp::QoS qos(1); // Call history depth constructor with size of 1
qos.transient_local(); // Use transient local 
 gear_pub_ = create_publisher<automotive_platform_msgs::msg::GearCommand>("as/gear_select", qos);
```